Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -115,8 +115,8 @@ IOS::ES::TicketReader CVolumeWiiCrypted::GetTicket() const
 
 IOS::ES::TMDReader CVolumeWiiCrypted::GetTMD() const
 {
-  u32 tmd_size;
-  u32 tmd_address;
+  u32 tmd_size = 0;
+  u32 tmd_address = 0;
```

Comment:
Oh, and the function should also return an empty `TMDReader` if the final `Read` call fails. This won't get rid of any warning, but it would be nice if you could fix it while you're at it.